src/app/complex/complex-results/complex-navigator-buttons/complex-navigator-buttons.component.css

```diff
@@ -16,7 +16,6 @@
 }
 
 .filters {
-  width: 120.5%;
```

Comment:
Why was this removed?

Reply:
As we moved the complex navigator into a div to be modified depending on the basket and the search result page, the buttons were not centered anymore with this width